src/promptflow-azure/promptflow/azure/_storage/cosmosdb/span.py

```diff
@@ -36,8 +39,8 @@ def __init__(self, span: SpanEntity, collection_id: str, created_by: dict) -> No
         self.context = span.context
         self.kind = span.kind
         self.parent_id = span.parent_id
-        self.start_time = span.start_time.isoformat()
-        self.end_time = span.end_time.isoformat()
```

Comment:
I think the problem here is that timezone information is lost, it may not be a good idea to simply add `Z` at the end.

Reply:
Adding `Z` means the time is UTC, then UX could render by customer's current time zone. Think is good enough

Reply:
Our telemetry is using UTC time zone.